pages/understanding-json-schema/reference/non_json_data.md

```diff
@@ -7,84 +7,128 @@ section: docs
 
 <Star label="New in draft 7" />
 
-JSON schema has a set of [keywords](../../learn/glossary#keyword) to describe and optionally validate
-non-JSON data stored inside JSON strings. Since it would be difficult to
-write validators for many media types, JSON schema validators are not
-required to validate the contents of JSON strings based on these
-keywords. However, these keywords are still useful for an application
-that consumes validated JSON.
+JSON schema has a set of [keywords](../../learn/glossary#keyword) to describe and optionally validate non-JSON data stored inside JSON strings that are stored or transmitted. Due to the difficulty in writing validators for all media types, JSON schema validators are not required to validate the contents of JSON strings based on these keywords. However, these keywords are useful for applications that consume validated JSON during the processes of encoding and decoding data.  
 
 <Keywords label="single: contentMediaType single: media; contentMediaType" />
 
-## contentMediaType
+## contentMediaType and contentEncoding
 
-The `contentMediaType` keyword specifies the MIME type of the contents
+The `contentMediaType` keyword specifies the MIME type of the content
```

Comment:
I struggled to understand what MIME means. I researched, and it says this (Multipurpose Internet Mail Extensions). Even though there's an RFC schema definition link after, it would be good to have the meaning there so we don't confuse the user. 

So it can be re-written as MIME (Multipurpose Internet Mail Extensions).

Reply:
"MIME type" is an old term that isn't really used any more. The more modern term is "media type". I suggest using that term instead.

Reply:
Agreed and adopted